doc/requirements/requirements.md

```diff
@@ -0,0 +1,306 @@
+MAP011 | Mapping.Data Model.Safety Zone.Intersection | 1 | May want to slow down (or even stop), for example, when approaching an intersection | Robots can slow down when coming to an intersection in order to ensure that they can see anyone coming and other people/robots can see them
+MAP012 | Mapping.Data Model.Lanes | 1 | Able to specify virtual lanes. Prefer the specified lanes (such as in a warehouse)
+MAP013 | Mapping.Data Model.Building Levels | 1 | Able to specify single and multi-level buildings
+MAP014 | Mapping.Data Model.Building Levels.Level Connecting Features | 1 | Able to specify level connecting features, such as elevators, stairways, ramps.
+MAP015 | Mapping.Data Model.Unknown Space | 1 | To be dynamically mapped? In global maps, it can be areas beyond the edge of the map, or areas within the center of the map for which we didn't have any observations during map building. In the current system, this is the "dark grey" portion of the map (free space is light grey, obstacles are black)
+MAP016 | Mapping.Data Model.Extensibility | 1 | *TODO: Map attributes. Layers concept?*
+
+### 2.5.2 Sensing
+
+The Navigation System requires dyanamic information about objects in its environment. This information is provided by a Sensory Subsystem that is outside the scope of the Navigation System. This information would typically be provided by a set of sensors on the robot and a sensor fusion capability. This section lists the requirements on the data that the Sensory Subsystem provides. 
+
+Id | Handle | Priority | Description | Notes
+-- | ------ | -------- | ----------- | -----
+SEN001 | Sensing | 1 | The Sensory Subsystem SHALL provide information about the dynamic environment of the robot | Info sufficient to carry out the Navigation System requirements
+SEN002 | Sensing.Latency | 1 | TODO
+SEN003 | Sensing.Environment Model | 1 | *TODO: What, specifically, should be in the environment model?*
+SEN004 | Sensing.Environment Model.Object Prediction | 1 | The Sensing Subsystem SHOULD provide an object prediction capability that projects the position of the objects into the future. | One of the biggest shortcomings of the current system is the inability to model/predict where obstacles will be in the future. This leads to collisions with other moving objects
+SEN005 | Sensing.Environment Model.Object Prediction.Time Horizon | 1 | *TODO: How far into the future should the object prediction work?*
+
+### 2.5.5 Localization
+
+The Navigation System requires the Robot's current pose, provided by an external Localization module. This section lists the requirements for the information from the Localization module.
+
+Id | Handle | Priority | Description | Notes
+-- | ------ | -------- | ----------- | -----
+LOC001 | Localization.Robot Pose | 1 | The Localization module MUST provide the robot's current pose to the Navigation System | This could be manual or as a result of automatic localization; the Navigation System wouldn't know either way 
```

Comment:
This won't be exact -- so it should also provide an estimate of the accuracy (current navigation stack can't take that into account, but future local planners certainly could). The current message is PoseWithCovariance to accomplish this in ROS1.

Reply:
Ok, I'll update.